apps/meteor/server/models/raw/Users.js

```diff
@@ -258,6 +258,112 @@ export class UsersRaw extends BaseRaw {
 		return this.findPaginated(query, options);
 	}
 
+	findPaginatedActiveUsersByRoomIdWithHighestRole(
+		searchTerm,
+		rid,
+		options,
+		searchFields,
+		extraQuery = [],
+		{ startsWith = false, endsWith = false } = {},
+	) {
+		if (options == null) {
```

Comment:
Why not making this the default value and update the callers to not pass null? 👀

Reply:
Done! btw this is the same we're doing on `findPaginatedByActiveUsersExcept`, so I changed it there too